### DISCUSSION TITLE
Editing Plots obtained by ORTools

### DISCUSSION THREAD

**Discussion by huseyinhzl** [ORTools4MTEX/ORTools#27] [General]
Hi all,

I am using MTEX 5.8.2 and ORTools 2.0.2 for parent grain reconstruction and I have two problems that I have not been able to solve by myself.

1.How can I remove the bounding box around the graphs that I have plottedwith MTEX? I have tried the box off and noFrame commands when I am plotting but it did not help, so I am wondering if there is a solution for this.

2.How can I change the line color in plotMap_variants command? Currently, it is black but I want to make it white so that my grains will be in a better look. I put an example image of variant plot
![image](https://user-images.githubusercontent.com/120036622/213549092-f2c7e4bf-b274-4203-af51-880fc9b77362.png)

Thank you so much

**Comment by frankNiessen** [comment]
Hi Huseyin, 

this worked for me:

```
plotMap_variants(job,'linewidth',1,'linecolor','w');
set(gca,'box','off');
```

Best wishes
Frank

**Reply by huseyinhzl** [reply]
Hi Frank,

Thank you for your suggestion. I don't know why the linecolor did not work for me at the beginning but, it is working now.  box off command only worked to remove the X2 and Y2 of the plot, yet I have solved it by set (gca, 'Visible', 'off');
